src/pages/Cal.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
```

Comment:
- [OPTIONAL] Instead of disabling `eslint` it will be nice to fix the issue, since `linter` are there to help us implement best practices. Please kindly fix the issue.